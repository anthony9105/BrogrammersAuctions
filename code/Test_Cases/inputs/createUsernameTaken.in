create
newUser123
close     