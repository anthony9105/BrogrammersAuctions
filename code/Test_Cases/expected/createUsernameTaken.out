Enter new username
Error.  Username is already in use.  Enter a new username